_posts/2021-01-19-bazel-4-0.md

```diff
@@ -0,0 +1,99 @@
+---
+layout: posts
+title: "Bazel 4.0"
+authors:
+  - communikit
+---
+
+Bazel 4.0 is a major release and Bazel’s first LTS release. It contains new features and backwards incompatible changes.
```

Comment:
"first LTS release" had a link in the Google doc, please keep that

Reply:
Done

Reply:
Done.